### DISCUSSION TITLE
Applicability of (data : ...) examples.

### DISCUSSION THREAD

**Discussion by Vincent-de-Comarmond** [Lisp-Stat/lisp-stat#20] [Q&A]
Hello

Firstly - thanks for the work. I appreciate it.
Secondly - I'm wondering about the `(data : ...)` examples in (see the [Data Frame](https://lisp-stat.dev/docs/tutorials/data-frame/) tutorial).

Look at the following lines (in [vgcars.lisp](https://github.com/Lisp-Stat/lisp-stat/blob/master/data/vgcars.lisp)):
```
(in-package #:ls-user)

(defdf vgcars (vega:read-vega vega:cars))
```

This means that the vgcars dataframe is loaded into the ls-user package (and not exported from it).
Which in turn means that if a user wants to create a system or package of their own and use `(lisp-stat:data : ...)` referring to the sample dataframe becomes very much non-transparent; the user has to do something like this:
`(lisp-stat:print-data ls-user::vgcars)` 

Does anybody feel like this is a situation worth improving?

**Comment by snunez1** [comment]
I think this is worth improving.  The `data` command is patterned after the one in R, where it's used to load datasets for tutorials and the like, but I see no reason to limit it to that.  Perhaps this is as simple as removing the `in-package` line, or possibly exporting it at the end.

This is a good beginner issue.  Is it something you're interested in working on?
